### DISCUSSION TITLE
tareas pendientes cuantas hay en total?

### DISCUSSION THREAD

**Discussion by rpajares** [Launch-X-Latam/MisionFrontEnd#203] [Q&A]
Hola, todo el mundo escribe muchas cosas de diferentes temas, no entiendo el proceso de esto, ya me perdi, alguien sabe donde puedo ver las tareas pendientes, sin que tenga que ver todo lo que escriben porque es confuso, no hay un panel personal, donde pueda ver mi avance o las cosas pendientes que tengo?, que hay que entregar y a quien se le entrega?, ojala alguien pueda leer esto.

**Comment by hanniafl** [comment]
Hola, yo tampoco se pero tu sabes en donde puedo ver las practicas que tenemos que realizar??

**Reply by snowingt** [reply]
hasta ahora solo tenemos la de html esa si tiene fecha que es el 4 del mes que viene , lo de la intro por lo que veo es algo asi como para tener nociones  de los pasos a seguir desde 0 pero igual ayuda tenerlo en la repo para ir creando el portafolio

**Reply by rpajares** [reply]
Entonces esa que dice abocat o algo asi, si uno quiere lo hace, si no no?, y alguien sabe que diferencia hay en que tengas tus proyectos aqui que en mi computador o mi hosting?

**Reply by EsthebanDiego** [reply]
Las practicas están en el repositorio de git de Launch X en [FrontEnd Mission](https://github.com/Launch-X-Latam/MisionFrontEnd), a lo que mencionaron es que creamos cada quien un repositorio en donde vamos subiendo las practicas, también que para poder pasar a la segunda parte de BackEnd tenemos que `subir por lo menos la mitad de las prácticas de FrontEnd`.
los videos grabados están en teams.  Hasta el momento estamos terminando la parte de HTML que es el segundo tema.
